modules/sbo-release-notes-1-1.adoc

```diff
@@ -0,0 +1,95 @@
+
+* *Service Binding* with the `binding.operators.coreos.com` API group.
+* *Service Binding (Spec API Tech Preview)* with the `servicebinding.io` API group.
++
+[IMPORTANT]
+====
+*Service Binding (Spec API Tech Preview)* with the `servicebinding.io` API group is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+[id="support-matrix-1-1_{context}"]
+== Support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Support matrix
+[cols="1,1",options="header"]
+|===
+| Feature | {servicebinding-title} 1.1
+| `binding.operators.coreos.com` API group
+| GA
+| `servicebinding.io` API group
+| TP
+|===
+
+[id="new-features-1-1_{context}"]
+== New features
+This section highlights what is new in {servicebinding-title} 1.1:
+
+* Service Binding Options
+** Workload resource mapping: Define exactly where binding data needs to be projected for the secondary workloads.
+** Bind new workloads using a label selector
+
+
+[id="fixed-issues-1-1_{context}"]
+== Fixed issues
+* Before this update, service bindings that used label selectors to pick up workloads did not project service binding data into the new workloads that matched the given label selectors. As a result, the Service Binding Operator could not periodically bind such new workloads. With this update, service bindings now project service binding data into the new workloads that match the given label selector. The Service Binding Operator now periodically attempts to find and bind such new workloads. link:https://issues.redhat.com/browse/APPSVC-1083[APPSVC-1083]
+
+
+[id="known-issues-1-1_{context}"]
+== Known issues
+* Currently, when you install the {servicebinding-title} in the single namespace installation mode, the absence of an appropriate namespace-scoped role-based access control (RBAC) rule prevents the successful binding of an application to a few known Operator-backed services that the {servicebinding-title} can automatically detect and bind to. In addition, the following error message is generated:
```

Comment:
This sentence is well-crafted, but its length makes it harder to interpret. Would it be possible to divide this into two or three sentences?

Reply:
Is there literally a "single namespace installation mode?"

Reply:
How about:
```suggestion
* There is currently a known issue with installing {servicebinding-title} in a single namespace. The absence of an appropriate namespace-scoped role-based access control (RBAC) rule prevents {servicebinding-title} from binding an application to a few known Operator-backed services that it can detect. When this happens, it generates the following error message:
```

Reply:
> Is there literally a mode called "single namespace installation mode?

Yes, there is :)

Reply:
The **successful** word is important here, because even if the binding happens it may not be successful as the application may not yet be connected to the database service. As a result, the pod fails to start and when you try to interact with the application, it returns errors. The user is aware of this detail and hence the need for the **successful** word. That way it is more specific.

Reply:
This looks accurate to me.